docs/modules/ROOT/pages/support/index.adoc

```diff
@@ -1,9 +1,13 @@
 = Support for {ProductName}
 
+
 To get support you can:
 
-* Email us - link:mailto:stonesoup@redhat.com[Stone Soup mailing list].
 
-* Contact with us in our Google Group link:https://groups.google.com/g/stonesoup[Google group: Stone soup]
+* Join us on Slack - join the link:https://dn.dev/DevNationSlack[DevNation workspace] and find us in the link:https://rhdevnation.slack.com/archives/C04LXT1EU7K[#software-supply-chain-security] channel.
+
+
+* Email us - link:mailto:sscs@redhat.com[sscs@redhat.com].
+
 
-* File a Jira ticket in the link:https://issues.redhat.com/projects/STONE[STONE project].
+* Select `Raise an issue` in the banner of any page in these docs.
```

Comment:
I think we should probably avoid saying "these docs". You usually don't want to talk about the documentation in your documentation

How about "Select `Raise an issue` on the top banner on any page"

Reply:
Oh I also just realized UI buttons should be bold, not monospace